README.md

```diff
@@ -228,6 +228,20 @@ postcss()
   })
 ```
 
+## `dependency` Message Support
+
+`postcss-import` adds a message to `result.messages` for each `@import`. Messages are in the following format:
+
+```
+{
+  type: 'dependency',
+  file: absoluteFilePath,
+  parent: fileContainingTheImport
+}
+```
+
+This is mainly for use by postcss runners that implement file watching.
```

Comment:
@michael-ciniawsky Just to confirm, the current version of `postcss-loader` listens for this message, correct?